specification/storage/resource-manager/Microsoft.Storage/preview/2020-08-01-preview/storage.json

```diff
@@ -2132,6 +2138,28 @@
         }
       }
     },
+    "EncryptionIdentity":{
```

Comment:
As we discussed in mail, as we don't want to expose this part in SDK, please remove it from swagger. (SDK is generated from swagger).

Reply:
@blueww  The EncryptionIdentity property is necessary in order for the user to be able to create a Storage Account with CMKL at creation time. Will update this thread after hearing back from PM.